File: api-ruby/app/models/todo_item.rb
class TodoItem
  include Mongoid::Document
  include Mongoid::Timestamps
  belongs_to :todo_list

  field :title, type: String
  field :url, type: String
  field :content, type: String
  field :completed, type: Boolean, default: false
  field :order, type: Integer

end

File: api-ruby/todo-ember/app/models/todo-item.js
import DS from 'ember-data';
const { attr, belongsTo } = DS;

export default DS.Model.extend({
  title: attr('string'),
  content: attr('string'),
  completed: attr('boolean'),
  order: attr('number'),
  url: attr('string'),

  todoList: belongsTo('todo-list')
});
